x509_auth/auth_backend.py

```diff
@@ -38,7 +38,7 @@ def is_X509_authenticated(request):
     Check how this user authenticated (did they use our backend?)
     """
     try:
-        return request.user.is_authenticated() and (
+        return request.user.is_authenticated and (
```

Comment:
With this (and related changes) I think this would mandate that the version of Django required be >=1.10.  The setup.py (updated in another commit) leaves this at <1.11.  Maybe I'm wrong about when the member was introduced.  I'm okay with a strict(er) Django requirement, but the requirement should match the code.  I'd rather not put in is_function() code.